guide/foundations/backups.md

```diff
@@ -0,0 +1,265 @@
+   image = "/assets/images/guide/foundations/backups/backup-table.png"
+   retina = "/assets/images/guide/foundations/backups/backup-table@2x.png"
+   alt-text = "Table of backup levels"
+%}
+
+<br>
+
+#### Level 0
+## Encrypted cloud backup 
+
+By storing the recovery-phrase in an online location that is encrypted (not in plain text) and hard to access by anyone other than yourself, you are protected against *loss of wallet*. This is the absolute minimum you should do and only takes a few minutes.
+
+{% include image.html
+   image = "/assets/images/guide/foundations/backups/cloudbackup.png"
+   retina = "/assets/images/guide/foundations/backups/cloudbackup@2x.png"
+   alt-text = "Encrypted cloud backup"
+%}
+
+**Do**
+- Use an encrypted password manager like 1Password, LastPass, iCloud Keychain
+- Alternatively, use a wallet application with automatic cloud backup
+
+**Don’t** 
+- Screenshot the recovery-phrase and save it in Google Photos, iCloud photos
+- Write down the recovery-phrase in plain text on your device, Google Docs, Dropbox, Notes etc.
+
+**Suitable for**
+- For small amounts (less than a months salary) 
+- When you can't wait to set up a new wallet until you are at home
+
+<br>
+
+#### Level 1
+## Single paper backup
+
+By storing the recovery-phrase written down on paper in your own home you have protected against *loss of wallet* and *remote theft*.
+
+
+{% include image.html
+   image = "/assets/images/guide/foundations/backups/paperbackup.png"
+   retina = "/assets/images/guide/foundations/backups/paperbackup@2x.png"
+   alt-text = "24 word paper backup"
+%}
+
+
+**Do**
+- Write the recovery-phrase clearly with the words numbered, using permanent ink on card, or a [template](create a template people can print) //ToDo
+- Put it in an envelope labelled ‘Important’
+- Store it in a location where it is unlikely to be found by a non-family member, (alternative locations include: inside a book, at the bottom of a jar filled with coffee beans, behind a painting, in the attic etc.)
+- Document or tell next-of-kin where the backup can be found
+
+**Don’t** 
+- Scribble the recovery-phrase on a piece of paper using a pencil and throw it in a drawer / stick it on the fridge 
+
+**Suitable**
+- For small amounts (less than a months salary)
+- When you are setting up a new wallet at home
+
+<br>
+
+#### Level 2
+## Single metal backup
+
+By storing the recovery-phrase on metal in your own home you have protected against *loss of wallet*, *remote theft* and *physical damage*. 
+
+The best metal backup solutions are resistant to most physical damage, including fire, acid and high pressure.
+
+
+{% include image.html
+   image = "/assets/images/guide/foundations/backups/metalbackup.png"
+   retina = "/assets/images/guide/foundations/backups/metalbackup@2x.png"
+   alt-text = "Metal backup"
+%}
+
+
+**Do** 
+- Order a dedicated metal backup product, recommendations can be found [here](https://jlopp.github.io/metal-bitcoin-storage-reviews/)
```

Comment:
for metal backups that you have to punch, it's also important that it is indented properly.

Reply:
Added point.